CADD.pm

```diff
@@ -115,9 +115,10 @@ sub new {
 
   my $params = $self->params_to_hash();
   my @files;
+  $self->{non_sv_ann_file} = 0;
   # Check files in arguments
   if (!keys %$params) {
-    warn "WARNING: Using snv or indels CADD annotation file with structural variant can increase run time exponentially.".
+    warn "WARNING: Using snv or indels CADD annotation file with structural variant can increase run time exponentially. ".
```

Comment:
Suggesting we remove the space at the end of this line

Reply:
This space is actually used for separator between two lines as they are in the same paragraph. Its printed lie this - 
```
Using snv=<file> and/or indels=<file> with structural variant can increase run time exponentially. Consider creating separate input files for SNV/indels and SV and use appropriate CADD annotation file.
```

Without space it would seem bad - 
```
Using snv=<file> and/or indels=<file> with structural variant can increase run time exponentially.Consider creating separate input files for SNV/indels and SV and use appropriate CADD annotation file.
```